Documentation/user-guides/getting-started.md

```diff
@@ -93,152 +65,13 @@ spec:
-
-```yaml mdox-exec="cat example/rbac/prometheus/prometheus-service-account.yaml"
-apiVersion: v1
-kind: ServiceAccount
-metadata:
-  name: prometheus
-```
-
-```yaml mdox-exec="cat example/rbac/prometheus/prometheus-cluster-role.yaml"
-apiVersion: rbac.authorization.k8s.io/v1
-kind: ClusterRole
-metadata:
-  name: prometheus
-rules:
-- apiGroups: [""]
-  resources:
-  - nodes
-  - nodes/metrics
-  - services
-  - endpoints
-  - pods
-  verbs: ["get", "list", "watch"]
-- apiGroups: [""]
-  resources:
-  - configmaps
-  verbs: ["get"]
-- apiGroups:
-  - discovery.k8s.io
-  resources:
-  - endpointslices
-  verbs: ["get", "list", "watch"]
-- apiGroups:
-  - networking.k8s.io
-  resources:
-  - ingresses
-  verbs: ["get", "list", "watch"]
-- nonResourceURLs: ["/metrics"]
-  verbs: ["get"]
-```
-
-```yaml mdox-exec="cat example/rbac/prometheus/prometheus-cluster-role-binding.yaml"
-apiVersion: rbac.authorization.k8s.io/v1
-kind: ClusterRoleBinding
-metadata:
-  name: prometheus
-roleRef:
-  apiGroup: rbac.authorization.k8s.io
-  kind: ClusterRole
-  name: prometheus
-subjects:
-- kind: ServiceAccount
-  name: prometheus
-  namespace: default
-```
-
-For more information, see the [Prometheus Operator RBAC guide]({{< ref "rbac" >}}).
-
-The Prometheus custom resource defines the characteristics of the underlying
-concrete StatefulSet (number of replicas, resource requests/limits , ...) as
-well as which ServiceMonitors should be included with the
-`spec.serviceMonitorSelector` field.
-
-Previously, we have created the ServiceMonitor object with the `team: frontend`
-label and here we define that the Prometheus object should select all
-ServiceMonitors with the `team: frontend` label. This enables the frontend team
-to create new ServiceMonitors and Services without having to reconfigure the
-Prometheus object.
-
-```yaml mdox-exec="cat example/user-guides/getting-started/prometheus-service-monitor.yaml"
-apiVersion: monitoring.coreos.com/v1
-kind: Prometheus
-metadata:
-  name: prometheus
-spec:
-  serviceAccountName: prometheus
-  serviceMonitorSelector:
-    matchLabels:
-      team: frontend
-  resources:
-    requests:
-      memory: 400Mi
-  enableAdminAPI: false
-```
-
-To verify that the instance is up and running, run:
-
-```bash
-kubectl get -n default prometheus prometheus -w
-```
-
-By default, Prometheus will only pick up ServiceMonitors from the current
-namespace. To select ServiceMonitors from other namespaces, you can update the
-`spec.serviceMonitorNamespaceSelector` field of the Prometheus resource.
-
 ## Using PodMonitors
 
-Instead of a ServiceMonitor, we can use a PodMonitor which doesn't require the
-creation of a Kubernetes Service. In practice, the `spec.selector` label tells
-Prometheus which Pods should be scraped.
+We can utilize a `PodMonitor` object to monitor the pods. The `spec.selector` label specifies which Pods Prometheus should scrape.
```

Comment:
"monitor the pods" seems vague to me, should we say "scrape metrics from the pods" instead?